src/main/haskell/kore/src/Kore/Attribute/Parser.hs

```diff
@@ -157,12 +175,14 @@ runParser Parser { getParser } (Attributes attrs) = do
     recordApplication
         attrMap
         Application
-            { applicationSymbolOrAlias
+            { applicationSymbolOrAlias =
+              SymbolOrAlias { symbolOrAliasConstructor
```

Comment:
The style guide requests indents that are multiples of 4, with some exceptions allowed. This is not one of them.